core/src/translate/function.rs

```diff
@@ -142,6 +155,14 @@ pub fn translate_function(sql_function: &SqlFunction) -> Result<Expr> {
         .collect::<Result<Vec<_>>>()?;
 
     match name.as_str() {
+        "CONCAT" => {
+            check_len_min(name, args.len(), 1)?;
+            let mut exprs: Vec<Expr> = vec![];
+            for arg in args {
+                exprs.push(translate_expr(arg)?);
+            }
```

Comment:
```suggestion
            let exprs = args
                .into_iter()
                .map(translate_expr)
                .collect::<Result<Vec<_>>>()?;
```

Reply:
1 Right now if NULL is entered => concat("abc", NULL) => NULL, null is returned (just like ||)
2. Right now concat requires at least 1 arg, and there is no maximum number of arguments.